app/src/main/java/org/mozilla/tv/firefox/ext/Locale.kt

```diff
@@ -2,7 +2,11 @@ package org.mozilla.tv.firefox.ext
 
 import java.util.*
 
-
+/**
+ * Returns true if the following is true for _any_ of the elements in [allowedLocales]:
```

Comment:
optional nit: this comment might be harder to read because of how closely it mirrors the code. Also, mirroring the code means the comment is liable to get outdated. However, I understand why communicating this behavior to the caller is important and I'm not sure there's a better solution. I wonder if this comment would be simpler (e.g. more declarative, don't have to wrap my head around a loop) in prose-like english.

Reply:
There's got to be a better way to describe this one, but we spent a considerable amount of time trying to find it without any luck.  This was the clearest we could manage.